lib/widgets/widgets/widget_details.dart

```diff
@@ -53,13 +51,11 @@ class WidgetDetails extends StatelessWidget {
       case WidgetTypes.AEM_HEALTHCHECK_WIDGET:
       case WidgetTypes.AEM_BUNDLE_INFO_WIDGET:
         return true;
-        break;
       case WidgetTypes.CHECKBOX_WIDGET:
+      case WidgetTypes.WORLD_CLOCK_WIDGET:
```

Comment:
To chyba możemy usunąć, jak nie będzie zdefiniowanych przypadków dla `CHECKBOX_WIDGET` `WORLD_CLOCK_WIDGET` to i tak wpadnie w `default` i zwróci false